Makefile

```diff
@@ -281,4 +281,21 @@ refresh-mapping-files:
 #
 #stop-silk-service:
 #	@ echo -e "Stop silk service"
-#	@ cd infra/silk/ && docker-compose down
\ No newline at end of file
+#	@ cd infra/silk/ && docker-compose down
+
+
+#-----------------------------------------------------------------------------
+# API Service commands
+#-----------------------------------------------------------------------------
+build-api:
+	@ echo -e "$(BUILD_PRINT) Build API services $(END_BUILD_PRINT)"
+	@ docker-compose -p ${ENVIRONMENT} --file infra/api/docker-compose.yml --env-file ${ENV_FILE} build --no-cache --force-rm
+	@ docker-compose -p ${ENVIRONMENT} --file infra/api/docker-compose.yml --env-file ${ENV_FILE} up -d --force-recreate
+
+start-api:
```

Comment:
call it by its name, pls: `identity_manager`, or whatever was the component.

Reply:
ID_MANAGER_API

Reply:
this target is meant to start all APIs (it may be others in the future)